### ISSUE TITLE
Is ETLbox under MIT License?

### ISSUE THREAD

**Issue by fabiocbr75** [etlbox/etlbox#91]
Hello,
I don't understand the license type?
In the root of the git project, there is the MIT License file, but in the ETLbox folder (where there is the source code), there is a completely different license.

Can you explain better? Is it or not under MIT License?

Thanks

**Comment by andreaslennartz** [comment]
Hello @fabiocbr75 ,
thanks for raising this issue. 
ETLBox was open source in the past, and licensed under MIT. But due to too few contributions I decided to change it to closed source. 
The code in this public repo only contains a small subset of the functionalities. So far, I have kept it as a reference for the forks, and "kept it alive" by copying some of the changes to the closed source version into this repo. 
 
ETLBox consists of 2 parts - ControlFlow tasks (e.g. for creating tables or running sql) and DataFlow tasks. Only the ControlFlow tasks are part of this repo. If you want to run data flow pipeline, you can download a test version of ETLBox (www.etlbox.net) to try it out. The only limitation of the test version is that you are not allowed to read or write more than 5000 rows in a data flow. 

But you are right, the licensing is confusing - I will probably remove the MIT license in the future and replace it with the current license.

To summarize it: If you are only looking for the ControlFlow part, then you can use this source code if you like. But if you want to create data flow pipelines with ETLBox, you need to use the test version (https://www.nuget.org/packages?q=etlbox). If you want to deploy your code in production, then you'll need a license. 

Let me know if this answers your question?
